README.md

```diff
@@ -76,10 +76,10 @@ target 'YourApp' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
-  use_flipper!
-  post_install do |installer|
-    flipper_post_install(installer)
-  end
+- use_flipper!
+- post_install do |installer|
+-   flipper_post_install(installer)
+- end
```

Comment:
We disable Flipper to workaround an known issue: https://github.com/facebook/react-native/issues/30836